### PR TITLE
update php-7.4 stable tag

### PR DESCRIPTION
**Pull request by samanthajayasinghe**

### CODE REVIEW

File: php/php74/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM php:7.4-rc-fpm
+FROM php:7.4.0-fpm
```
Comment (derschatta) [inline php/php74/Dockerfile]: shall we make it `php:7.4`?

Comment (samanthajayasinghe) [reply php/php74/Dockerfile]: sure will do ..

Comment (samanthajayasinghe) [reply php/php74/Dockerfile]: done

Comment (derschatta) [reply php/php74/Dockerfile]: 👍

**Comment by derschatta** [comment]
I just realised @samanthajayasinghe  can you add PHP 7.4 to the travis.yml file? I don't think it's there so it will not be picked up by TravisCI

**Comment by samanthajayasinghe** [comment]
@derschatta added php-7.4 into Travis as well

**Pull request merged**